docs/web-console-guide/master.adoc

```diff
@@ -233,10 +233,11 @@ include::topics/mta-web-controlling-task-order-with-task-manager.adoc[leveloffse
 
 // include::topics/mta-web-creating-custom-migration-targets.adoc[leveloffset=+2]
 
```

Comment:
If you mean to delete it, go ahead. The file is in a previous branch, right? Double-check with Andy or Radhika, if you have any question.